### DISCUSSION TITLE
(Unsure) System cleaning

### DISCUSSION THREAD

**Discussion by anonimo82** [doublecmd/doublecmd#1826] [Suggestions]
Hello everyone.
As in title, please, add the feature to clean the host system by removing temp files, registry values if in Windows, or cache memory when in other OS. I'm unsure if this is in scope with DoubleCMD, if this is not, please tell me and please, don't consider this.
As an example, I can quote BleachBit (https://www.bleachbit.org/)

Thanks in advance for any help.
Regards, have a nice day.

**Comment by JonnyTech** [comment]
Out of scope for a file manager

**Comment by anonimo82** [comment]
Thanks for telling, excuse me.
